dev_guide/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,115 @@
+[[dev-guide-jenkins-pipeline]]
+= Jenkins Pipeline
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+
+== Overview
+
+Jenkins Pipelines give you fine-grained control over building, deploying, and promoting your applications on OpenShift.
+Using a combination of the Jenkins Pipeline Build Strategy, Jenkinsfiles, and the OpenShift DSL _(provided by the OpenShift
+Jenkins Client Plugin)_, you can create advanced build/test/deploy/promote Pipelines for any scenario.
+
+[[openshift-jenkins-Pipeline-plugin]]
+
+== OpenShift Jenkins Client Plugin
+
+The https://github.com/openshift/jenkins-client-plugin[OpenShift Jenkins Client Plugin] must be installed
+on your Jenkins master so that the OpenShift DSL will be available to use within the JenkinsFile for your
```

Comment:
fluent DSL

though maybe not every instance of DSL needs to become `fluent DSL` ... assuming everyone agrees on injecting `fluent` ....as long as it is emphasized somewhere in the doc, I'm good.

Reply:
Agreed, see comment at top.

Reply:
yep ... cool deal